inz-demo/description.xml

```diff
@@ -1,6 +1,6 @@
 <SchemeManager version="7">
 	<Id>inz-demo</Id>
-	<Url>https://demo.irma.nz/inz-demo/</Url>
+	<Url>https://irma-scheme.local.internetnz.nz</Url>
```

Comment:
any reason to change this to local address?

Reply:
I am testing on my local and it was committed while I am changing things. I will leave this open to not forget to revert and sign again before the merge.